modules/private-clusters-about-gcp.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/
+
+[id="private-clusters-about-gcp_{context}"]
+= Private clusters in GCP
+
+To create a private cluster on Google Cloud Platform (GCP), you must provide an existing private VPC and subnets to host the cluster. The installation program must also be able to resolve the DNS records that the cluster requires. The installation program configures the Ingress Operator and API server for only internal traffic.
```

Comment:
> To create a private cluster on Google Cloud Platform (GCP), you must provide an existing private VPC and subnets to host the cluster.

I think the installer may create private networks if none are provided. Users would then need to vpn to the installer-created network.

Reply:
The network and subnets must be provided by the user as there is no way for the installer to automatically access the cluster api in a private network it just provisioned (and therefore wait-for bootstrap-complete will always fail).